snippets/scripts/jira-components-to-csv/README.md

```diff
@@ -0,0 +1,11 @@
+# Usage
+
+This Python script will grab all Jira components from a user-defined selection of Jira projects.
```

Comment:
1. I think we should limit the maximum number of projects one can add. 
2. Specify the total number of components the script can process in one execution - MAX 1000. I understand that the underlying assumption here is that no customer would have more than 100 components, however, we have come across a customer with 1000s of jira components previously.

Reply:
How many projects? Maybe 100?

Reply:
Works